src/main/java/duke/command/ViewCommand.java

```diff
@@ -0,0 +1,41 @@
+package duke.command;
+
+import java.time.LocalDate;
+import java.time.format.DateTimeParseException;
+
+import duke.DukeException;
+import duke.Storage;
+import duke.TaskList;
+import duke.Ui;
+
+/**
+ * Represents a command to see all the tasks on a certain date.
+ */
+public class ViewCommand extends Command {
+    /** The date on which you want to view the tasks for. */
+    private String date;
+
+    /**
+     * Creates a new view command with the specified date.
+     * @param date The date on which you want to view the tasks for.
+     */
+    public ViewCommand(String date) {
+        this.date = date;
+    }
+
+    /**
+     * View the tasks on the date in the specified task list.
```

Comment:
Comment is missing the empty line between description and parameter.

Reply:
Noted!